src/libraries/System.Net.Sockets/tests/FunctionalTests/TcpListenerTest.cs

```diff
@@ -44,6 +44,30 @@ public void Active_TrueWhileRunning(int ctor)
             Assert.False(listener.Active);
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsWindows))]
+        public void AllowNatTraversal_NotStarted_SetSuccessfully()
+        {
+            var listener = new TcpListener(IPAddress.Loopback, 0);
+            listener.AllowNatTraversal(true);
```

Comment:
Shouldn't we be disposing of the listener, here as well as in the below tests?

Reply:
It doesn't implement IDispose.

Reply:
In this case "disposing" means Stop'ing.  That's what disposes of the underlying socket.

Reply:
Ah, yes. Stop is called in all other tests. I will add here as well.